expressEngine/user/service/user.js

```diff
@@ -49,6 +49,16 @@ class Service extends BaseService {
 
     return user;
   }
+
+  delete = async (id) => {
```

Comment:
@adilsuccessive Pass object, instead of id to make it generic for deleteOne or deleteMany. You can use a flag like isDeleteOne = true. If false, following function can act as deleteMany

Reply:
Done with the change